src/Components/ArtistFilter/Artworks.tsx

```diff
@@ -0,0 +1,147 @@
+import * as React from "react"
+import {
+  ConnectionData,
+  createPaginationContainer,
+  graphql,
+  RelayPaginationProp,
+} from "react-relay"
+import { Subscribe } from "unstated"
+
+import { FilterState } from "./state"
+import ArtworkGrid from "../ArtworkGrid"
+
+interface Props extends RelayProps {
+  relay?: RelayPaginationProp
+  artistID: string
+}
+
+const PageSize = 10
```

Comment:
This is likely taken from elsewhere in codebase but its more idiomatic to see constants like this expressed as 

```javascript
const PAGE_SIZE = 10
```